Packs/BitcoinAbuse/Integrations/BitcoinAbuse/BitcoinAbuse.yml

```diff
@@ -129,7 +128,13 @@ script:
     description: Reports an abuser to Bitcoin Abuse service. 'abuse_type_other' field is
       required when 'abuse_type' is other
     name: bitcoinabuse-report-address
-  dockerimage: demisto/python3:3.8.6.14516
+  - arguments:
+    - defaultValue: "50"
+      description: The maximum number of results to return. The default value is 50.
```

Comment:
This is added automatically when you set a default
<img width="665" alt="Playground 2021-01-17 15-24-43" src="https://user-images.githubusercontent.com/44666568/104844304-417fb200-58d8-11eb-9c2b-5f59f7718c98.png">

```suggestion
      description: The maximum number of results to return. 
```